### PR TITLE
Update typography; fixes #2093

### LINKED ISSUE #2093: Typography classes & styles
Below are the font sizes, weights, and letter-spacing for the classes defined here:
https://material.io/develop/web/components/typography/

(Specify rems; px are for reference.)

class | rem | px | weight | line-height | letter-spacing | other
--- | --- | --- | --- | --- | --- | ---
Headline 1 | 2.822rem | 45.159 | 300 | 1.2em | -.015em
Headline 2 | 1.575rem | 25.192 | 300 | 1.33em | 0
Headline 3 | 1.383rem | 22.127 | 300 | 1.33em | 0
Headline 4 | 1.215rem | 19.436 | 400 | 1.45em | 0.007em
Headline 5 | 1.138rem | 18.215 | 400 | 1.55em | 0.005em
Headline 6 | 1.067rem | 17.072 | 400 | 1.6em | 0.01em
Subtitle 1 | 0.937rem | 14.995 | 300 | 1.6em | .015em
Subtitle 2 | 0.878rem | 14.053 | 500 | 1.75em | .02em
Body 1 | 1rem | 16px | 400 | 1.6em | 0
Body 2 | 0.878rem | 14.053 | 400 | 1.6em | .015em
BUTTON | 0.878rem | 14.053 | 500 | 2.25**rem** | 0.09em | uppercase
Caption | 0.772rem | 12.344 | 400 | 1.6**rem** | 0.33em
OVERLINE | 0.678rem | 10.842 | 500 | 2em | .166em | uppercase

## assign classes

Some specific class assignments are needed to override default sizes. 

element | assign class
--- | ---
window title `<h2>` | Heading 6
companion window `<h3>` | Subtitle 1
`<dt>` |  Subtitle 2
`<dd>` | Body 1 
"Current item" etc. labels in About panel | OVERLINE
window dialog `<h3>` | Heading 2 (we don't have any of these yet)

typography

### PR DESCRIPTION
**Pull request by cbeer**
I've tried to reconcile the table in #2093 with the theme defaults in https://material-ui.com/customization/default-theme/, and only adding the difference to our theme. Hopefully I translated everything correctly...

### CODE REVIEW

File: src/config/settings.js [hunk 1]
```diff
@@ -21,6 +21,99 @@ export default {
       },
     },
     typography: {
+      fontSize: 16,
+      body1: {
+        fontSize: "0.878rem",
```
Comment (jkeck) [inline src/config/settings.js]:
```suggestion
        fontSize: "1rem",
```

File: src/config/settings.js [hunk 2]
```diff
@@ -21,6 +21,99 @@ export default {
...
+        lineHeight: "2.25rem",
+        textTransform: "uppercase",
+      },
+      caption: {
+        fontSize: "0.772rem",
+        letterSpacing: "0.33em",
+        lineHeight: "1.6rem",
+      },
+      body1Next: {
+        fontSize: "0.878rem",
```
Comment (jkeck) [inline src/config/settings.js]:
```suggestion
        fontSize: "1rem",
```

**Comment by jvine** [comment]
One problem from a (now fixed) typo in my table:

Button should have letter-spacing of 0.09em (not 0.9em).

**No**
Screen Shot 2019-03-08 at 12 29 05 PM

**Yes**
Screen Shot 2019-03-08 at 12 28 52 PM

File: src/config/settings.js [hunk 3]
```diff
@@ -21,6 +21,99 @@ export default {
...
+        lineHeight: "1.6em",
+      },
+      body2Next: {
+        fontSize: "0.878rem",
+        letterSpacing: "0.015em",
+        lineHeight: "1.6em",
+      },
+      buttonNext: {
+        fontSize: "0.878rem",
+        letterSpacing: "0.9em",
```
Comment (cbeer) [inline src/config/settings.js]:
```suggestion
        letterSpacing: "0.09em",
```

File: src/config/settings.js [hunk 4]
```diff
@@ -21,6 +21,99 @@ export default {
...
+        lineHeight: "1.6em",
+      },
+      body2: {
+        fontSize: "0.878rem",
+        letterSpacing: "0.015em",
+        lineHeight: "1.6em",
+      },
+      button: {
+        fontSize: "0.878rem",
+        letterSpacing: "0.9em",
```
Comment (cbeer) [inline src/config/settings.js]:
```suggestion
        letterSpacing: "0.09em",
```

**Comment by aeschylus** [comment]
Looks great at the demo link.

**Pull request merged**